readme.md

```diff
@@ -0,0 +1,147 @@
+| [`@statechannels/node-provider`](/packages/node-provider) | [![npm](https://img.shields.io/npm/v/@statechannels/node-provider.svg)](https://www.npmjs.com/package/@statechannels/node-provider) | Wrapper around API calls to a Node |
+
+**Utilities**
+
+| Package | Version | Description |
+| --- | --- | --- |
+| [`@statechannels/types`](/packages/types) | [![npm](https://img.shields.io/npm/v/@statechannels/types.svg)](https://www.npmjs.com/package/@statechannels/types) | Shared type declarations |
+| [`@statechannels/typescript-typings`](/packages/typescript-typings) | [![npm](https://img.shields.io/npm/v/@statechannels/typescript-typings.svg)](https://www.npmjs.com/package/@statechannels/typescript-typings) | Shims for external library typings |
+
+## Contributing
+
+- **Join us in [Discord][statechannels-discord-url]** to help with development or to hang out with some ethereum hackers :)
+- **[Create a new issue](https://github.com/statechannels/monorepo/issues/new)** to report bugs
+- **[Fix an issue](https://github.com/statechannels/statechannels/issues?state=open)**. statechannels is an [Open Source Project](.github/CONTRIBUTING.md)!
+
+### Ethereum Networks
+
+The Playground currently only supports the Kovan network.
+
+### Installing dependencies
+
+**Make sure you have Yarn v1.19.0 installed**. For easy management of specific Yarn versions, we recommend using [Yarn Version Manager (YVM)](https://github.com/tophat/yvm).
+
+To install the dependencies:
+
+```shell
+yarn
+```
+
+### Building packages
+
+It is recommended (optional) that you installed `solc` compiler binary on your machine ([installation guide](https://solidity.readthedocs.io/en/latest/installing-solidity.html#binary-packages)) and specify the following in your `.env` under `cf-funding-protocol-contracts` and `apps`
+
+```shell
+NATIVE_SOLC=true
+```
+
+To build all packages:
+
+```shell
+yarn build
+```
+
+### Clean
+
+To clean all packages:
+
+```shell
+yarn clean
+```
+
+### Lint
+
+To lint all packages:
+
+```shell
+yarn lint
+```
+
+To also apply automatic fixes:
+
+```shell
+yarn lint:fix
+```
+
+### Tests
+
+To run all tests:
+
+```shell
+yarn test
+```
+
+To run E2E tests:
+
+- If you're using E2E tests for the first time or have done recent changes to the code, run:
+
+```shell
+yarn test:e2e:setup
+```
+
+- If you're re-running tests or writing new scenarios, run:
+
+```shell
+NODE_EXTENDED_PRIVATE_KEY="..." yarn test:e2e
+```
+
+See the ["Funding the Hub Account for Playground Testing"](./packages/simple-hub-server/README.md) section on the Simple Hub Server package for more information about `NODE_EXTENDED_PRIVATE_KEY`.
+
+### Patches
+
+We generate patches using the [`patch-package`](https://github.com/ds300/patch-package) tool with a workaround described in [this issue](https://github.com/ds300/patch-package/issues/42#issuecomment-435992316) for `yarn` workspaces.
```

Comment:
Perhaps we should discuss this approach as a part of our wider strategy on maintaining dependencies.

Reply:
As discussed, we should remove `patch-package` until such time as we want to fix bugs in external dependencies (when we can put it back in).